app/assets/javascripts/app/views/unblock_person_view.js

```diff
@@ -0,0 +1,29 @@
+// @license magnet:?xt=urn:btih:0b31508aeb0634b347b8270c7bee4d411b5d4109&dn=agpl-3.0.txt AGPL-v3-or-Later
+
+app.views.UnblockPerson = app.views.Base.extend({
+  templateName: "unblock_person",
+
+  events: {
+    "click #unblock_user_button": "unblock"
+  },
+
+  initialize: function(opts) {
+    _.extend(this, opts);
+    this.parent = opts.parent;
+  },
+
+  unblock: function() {
+    var unblock = this.person.unblock();
+    var that = this;
+
+    unblock.fail(function() {
+      app.flashMessages.error(Diaspora.I18.t("unblock_failed"));
+    });
+
+    unblock.done(function() {
+      that.parent._populateHovercard();
```

Comment:
Now that this method is called outside of the class it should probably drop the leading underscore since it is no longer private use.  The method itself assumes that parent has been set with a parent that has the href attr which is only true if the parent used already went through the showHovercardOn: _.debounce path.  It seems any time someone would use it that it would have been through that path already but if want to do some more defensive programming could catch/check that too perhaps.